astropy/units/physical.py

```diff
@@ -339,10 +354,7 @@ def __repr__(self):
         return f"PhysicalType({names})"
 
     def __str__(self):
-        if len(self._physical_type) == 1:
-            return self._physical_type_list[0]
-        else:
-            return self._name_string_as_ordered_set()
+        return ", ".join(self._physical_type_list)
```

Comment:
If we're dropping the braces, should we make this `"/".join(...)` so that in principle it can be read back?

Reply:
Is there any expectation that ``__str__`` can be read back? I thought that's what ``__repr__`` is for

Reply:
Agreed, just that we accept `/` on input, so it seems perhaps more logical to re-use that. I also like that it is commonly used to mean "or", while with the `,` the braces helped to make that clear.

Reply:
Clarity and round-tripping are definitely good things. I don't have a strong opinion whether "/" or "," is superior as they have a similar set of advantages and disadvantages, depending on use case.

Reply:
It would be worthwhile to have operations like `u.Pa.physical_type == str(u.Pa.physical_type)` work as expected, so I'll change it to `/`.  

We'd have to do a bit more to make it a full round trip, since the `/` separated physical types are allowed in `def_physical_type` and `PhysicalType` and with operators like `==`, but not in `get_physical_type`.  This was necessary for the former three cases since there was a previously a `str` physical type named `"momentum/impulse"` and we wanted to maintain backward compatibility, but this wasn't a concern for `get_physical_type` since it previously returned `"momentum/impulse"` instead accepting it as an argument.  Adding the full round trip is out of scope of this PR but it's something that could be added in later.